react-app/src/components/Profile/index.js

```diff
@@ -52,11 +52,11 @@ export default function Profile() {
           />
```

Comment:
I cannot for the life of me figure out how to get this button to style back down to the same size as the NavLinks above it :()